### ISSUE TITLE
UI accessibility or using something accessible to screen readers?

### ISSUE THREAD

**Issue by brightening-eyes** [DISTRHO/DPF#444]
hello,
based on what I know, cairo is not accessible, especially on windows. another big problem is that even the raw openGL stuff are completely graphical and are not accessible to screen reader users (blind and visually impaired).
the thing is, maybe something like [this](https://github.com/accesskit/accesskit) can be used, (which is mostly supported on windows and is developed in rust), or, is there another alternative that one can invent ui stuff? either by means of wxWidgets or something accessible?
thanks.

**Comment by falkTX** [comment]
honestly this is outside the scope of the dpf project.
from what I have seen the folks at juce tackling this, it is a huge amount of work, not something I wish to delve into due to its complexity and easy to get wrong. requires coordination with the target audience too.

if you care about accessibility you shouldnt be using dpf, or alternatively be prepared to do a lot of ground work first.

for the moment it is simply not planned as an official feature, you are on your own for this.
